### ISSUE TITLE
Save parameter file when training a network

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#12]
Parameters include:
- descriptor dimensionality
- learning rate
- loss type

We should store everything we need to build and run the network.
